modules/connatixBidAdapter.js

```diff
@@ -308,6 +330,21 @@ export const spec = {
       params['us_privacy'] = encodeURIComponent(uspConsent);
     }
 
+    window.addEventListener('message', function handler(event) {
+      if (!event.data || event.origin != 'https://cds.connatix.com') {
+        return;
+      }
+
+      this.removeEventListener('message', handler);
+
+      event.stopImmediatePropagation();
```

Comment:
Can you move this in the  `if (!response.optout && response.ids) ` ? Would be more specific

Reply:
but wouldnt i want this to happen regardless of the response? in case it doesnt get inside that if